components/webflow/actions/create-collection-item/create-collection-item.mjs

```diff
@@ -1,53 +1,59 @@
-// legacy_hash_id: a_q1ioGv
-import { axios } from "@pipedream/platform";
+import webflow from "../../webflow.app.mjs";
 
 export default {
   key: "webflow-create-collection-item",
-  name: "Create Item",
+  name: "Create Collection Item",
   description: "Create new collection item",
-  version: "0.1.1",
+  version: "0.2.1",
```

Comment:
I don't really see the reason on changing the `MINOR` version to `2` I would rather change the `PATCH` version right? like
```suggestion
  version: "0.1.2",
```
the same goes to the rest of components that were modified